src/js/components/AppComponent.jsx

```diff
@@ -78,8 +81,8 @@ var AppComponent = React.createClass({
         <td className="text-right health-bar-column">
           <AppHealthComponent model={model} />
         </td>
-        <td className="text-right">
-          <AppStatusComponent model={model} />
+        <td className="text-right actions">
+          ...
```

Comment:
Let's

```
<td className="text-right actions">&hellip;</td>
```

No line break if it fits, and let's use the proper HTML entity if available!